readinglist/resource.py

```diff
@@ -287,16 +403,27 @@ def collection_get(self):
 
         filters = self._extract_filters(self.request.GET)
         sorting = self._extract_sorting(self.request.GET)
-        records = self.db.get_all(filters=filters,
-                                  sorting=sorting,
-                                  **self.db_kwargs)
+        pagination_rules, limit = self._extract_pagination_rules_from_token(
+            self.request.GET)
```

Comment:
Since ``_extract_*`` methods have ``self`` maybe we could skip passing self.request.GET in arguments